### DISCUSSION TITLE
Google pagespeed & Oqtane prerender

### DISCUSSION THREAD

**Discussion by thienvc** [oqtane/oqtane.framework#2142] [General]
I use https://pagespeed.web.dev/ to test oqtane.org, The results returned are very bad, is this problem a priority to solve?

**Comment by sbwalker** [comment]
Are you asking if I intend to optimize the oqtane.org website design and content? The current site was designed for desktop... ie. the majority of the site content is rendered on a single page and the navigation menu simply scrolls to anchor tag locations within the page... and there are many images on the home page and some of them are very large - especially the rotating banner images at the top. This approach is fine for desktop but is obviously not optimal for mobile where you would want to minimize the amount of content as well as the size of the images. But as I said, the current site design is not focused on mobile.

The https://pagespeed.web.dev/ results are actually fairly decent for Desktop:

![image](https://user-images.githubusercontent.com/4840590/164257344-37cd06bd-7b7f-40f7-a08b-57213b166a37.png)

**Comment by thienvc** [comment]
![image](https://user-images.githubusercontent.com/1115393/164370637-bdcbfea7-827c-4d3f-b216-dc27792d9b92.png)
![image](https://user-images.githubusercontent.com/1115393/164370673-03515ade-134e-4303-916e-6c086adaa279.png)

wow, why is it so bad on my computer?
Maybe it's because I'm in Vietnam, so I'm far from the server

**Reply by Loclhh** [reply]
Dear @thienvc,

I'm in Viet Nam too and living in HCM. Which city are you now? Could have your email for personal contact?

Thank you!

**Reply by thienvc** [reply]
> Dear @thienvc,
> 
> I'm in Viet Nam too and living in HCM. Which city are you now? Could have your email for personal contact?
> 
> Thank you!

Yes, im from Vinh City (NgheAn Province), my email: it.thienvc@gmail.com

**Reply by Loclhh** [reply]
hi Thien,

I had sent a email to you. Please read it. Glad to be made friend with you.

Loc

**Reply by oqtanecms** [reply]
> 

I am in China,both desktop and mobile, the results are same bad and need to be optimized.

**Reply by qunshen** [reply]
When you test website cross-continent, result may be affect by network issues instread of web server.  So the test results may not be valid in  this case.
